src/cairo_dependencies_graph/import_parser.py

```diff
@@ -1,18 +1,33 @@
-from starkware.cairo.lang.compiler.ast.code_elements import CodeElementFunction, CodeBlock
+from starkware.cairo.lang.compiler.ast.code_elements import CodeElementFunction, CodeBlock, CodeElementImport
 from starkware.cairo.lang.compiler.ast.visitor import Visitor
 
 
 class ImportParser(Visitor):
 
     def __init__(self):
         super().__init__()
+        self.extracted_imports
```

Comment:
`self.extracted_imports = []`

Otherwise it doesn't work (at least on my side, should it work though?)

Reply:
right, missed that thanks